src/main/java/com/taboola/rest/api/RestAPIClient.java

```diff
@@ -1,5 +1,7 @@
 package com.taboola.rest.api;
 
+import okhttp3.Interceptor;
```

Comment:
@paul-hsieh we wouldnt like to expose OkHttp.
IF something is missing in the existing CommunicationInterceptor lets extend it instead.